app/models/comfy/cms/page.rb

```diff
@@ -97,7 +97,7 @@ def published_at
     elsif scheduled?
       scheduled_on
     elsif ever_been_published?
-      revisions.detect { |revision| revision.data[:event].include?('published') }.created_at
+      revisions.detect { |revision| Array(revision.data[:event]).include?('published') }.created_at
```

Comment:
Perhaps pull it out to a named method then to make it a bit more obvious (to dummies like me at least ;) ) what the intention is?